src/background/telemetry/telemetry-client-provider.ts

```diff
@@ -15,6 +15,7 @@ import { TelemetryClient } from './telemetry-client';
 import { TelemetryLogger } from './telemetry-logger';
 
 export const getTelemetryClient = (
+    applicationName: string,
```

Comment:
curiousity question: Is it intentional to make the `applicationName` parameter as the first param that gets passed to the `getTelemetryClient`?

Reply:
yes... but it's opinionated... `applicationName` is a `string` i.e. a primitive type, while all the other params are instances of some class/type we have define elsewhere